Public/Src/Cache/ContentStore/Distributed/NuCache/FlushableCache.cs

```diff
@@ -114,6 +128,8 @@ private void PerformFlush(OperationContext context)
                     _cache = new ConcurrentBigMap<ShortHash, ContentLocationEntry>();
                 }
 
+                stopwatch.Start();

```

Comment:
Just curious why you've decided to exclude locking from the duration?

Reply:
Because I am only interested in the actual transaction time and RocksDb writing time. Also, the lock:
* Is required for correctness, I couldn't optimize it out even if it was an issue.
* Although it is contended, I'm relying on it being fair; if it wasn't, we'd probably never see a flush happen.
* If really needed (i.e. flush time + cleanup time don't add up to the total time taken by the flush operation), then it can be added later.